Source/WebCore/dom/ElementTextDirection.h

```diff
@@ -45,6 +45,8 @@ enum class TextDirectionState : uint8_t {
     Undefined,
 };
 
+enum class TextDirectionUpdateReason : bool { Attribute, DOM };
```

Comment:
I don't have a great suggestion for alternative names, but an Attribute change is also a DOM change fwiw

Reply:
I added a new function named `textDirectionStateChanged()` and I removed this enumeration.